### DISCUSSION TITLE
Feature request: TalosOS

### DISCUSSION THREAD

**Discussion by lucawen** [mysticaltech/terraform-hcloud-kube-hetzner#1329] [Ideas]
I see that was created a lot of effort to make some changes for microos. But I think, maybe using TalosOS can be a better choice? Because of his configuration style. Im not an expert in both approaches, but I think that we can start to discuss about it !

Thanks for all the effort on this project :)

**Comment by otavio** [comment]
Take a look at https://github.com/hcloud-talos/terraform-hcloud-talos; might help.

**Comment by mysticaltech** [comment]
Thanks for sharing folks, I have been considering Talos for some time. Good to see someone already did the jump!

**Comment by mysticaltech** [comment]
Since this has already been done by someone else and is actively maintained, this is not planned for the moment. However very soon we will be moving our nodes to LeapMicro which is much more stable and enterprise ready.
